src/libopensc/card-piv.c

```diff
@@ -1610,7 +1585,7 @@ static int piv_general_mutual_authenticate(sc_card_t *card,
 	/* get the encrypted nonce */
 	r = piv_general_io(card, 0x87, alg_id, key_ref, sbuf, p - sbuf, rbuf, sizeof rbuf);
 
- 	if (r < 0) goto err;
+	if (r < 0) goto err;
```

Comment:
clang-format suggested the following change:


```
-	if (r < 0) goto err;
+	if (r < 0)
+		goto err;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-piv.c -->